django_ses_plus/models.py

```diff
@@ -5,7 +5,7 @@
 from django.db import models
 from django.template.loader import render_to_string
 from django.utils import translation
-from django.utils.translation import ugettext_lazy as _
+from django.utils.translation import gettext_lazy as _
```

Comment:
Deprecated in Django 3.0, https://docs.djangoproject.com/en/4.1/releases/3.0/#id3